catalyst/dl/experiment/config.py

```diff
@@ -479,6 +478,15 @@ def _process_callbacks(callbacks: OrderedDict) -> None:
         """
         for callback in callbacks.values():
             if isinstance(callback, CheckpointCallback):
+                # use resume only for first stage
+                _resume = self._config["args"].pop("resume", None)
```

Comment:
let's move it here
https://github.com/catalyst-team/catalyst/blob/master/catalyst/core/callbacks/checkpoint.py#L553
what do you think about it?

Reply:
Current implementation updates arguments of CheckpointCallback.
Could you please describe more about what's exactly need to move to callback?

Reply:
I mean, based on [this code](github.com/catalyst-team/catalyst/blob/master/catalyst/core/callbacks/checkpoint.py#L548#L558) `CheckpointCallback` already uses arguments from state/runner like `resume` and `resume-dir`
so, could we just propagate them there? what do you think about it?